tests/radiation/radiationBoost.test.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2022 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of the GNU Affero General Public License v3.0.
+ * You should have received a copy of the GNU Affero General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY                           
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES                          
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT                           
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,                                
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED                          
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;                               
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER                              
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN                         
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *  
+ *
+ */
+
+const validation = require('./../../references/validationFunctions/radiation/radiationBoost.js');
+const universalTest = require('../universal');
+const loadObjects = require('../loadObjects');
+
+// load in all fields with entries prepopulated to null
+const radiation = require('../constructDummyData').getSchemaDummy('radiation');
+
+const myUnitTests = {
+    "reference_radiation_treatment_id": [
+        [
+            'reference_radiation_treatment_id submitted when radiation boost was given.',
+            true,
+            loadObjects(radiation,
+                {   
+                    "radiation_boost": "yes",
+                    "reference_radiation_treatment_id": "tr2"
```

Comment:
Will there ever be a case when treatment ID is "unknown" or other generic values? if so, is that valid?

Reply:
@justincorrigible you pointed out a valid use case. However based on the current validation rule, if indicating `radiation_boost` as `yes` while treatment ID is "Unknown", the program will need to apply exception on field `reference_radiation_treatment_id`.